dask/array/tests/test_fft.py

```diff
@@ -195,3 +195,9 @@ def test_wrap_fftns(modname, funcname, dtype):
         wfunc(darr2c, (darr2c.shape[0] - 1, darr2c.shape[1] - 1), (0, 1)),
         func(nparrc, (nparrc.shape[0] - 1, nparrc.shape[1] - 1), (0, 1))
     )
+
+
+@pytest.mark.parametrize("n", list(range(1, 10)))
+@pytest.mark.parametrize("d", [1.0, 0.5, 2 * np.pi])
```

Comment:
I tried handling the `AssertionError` in the test so we could print the parameters, but it did not go well for some reason.

Have cut down the range to 5 values. Hopefully that cuts down visual noise a little bit. Should also provide sufficient coverage. Would that work for you?